src/main/webapp/app/views/inputtype/input-select.html

```diff
@@ -9,7 +9,7 @@
     <li class="select-option"
       role="menuitem"
       ng-click="fieldValue.value = word.name; fieldProfileForm.$dirty = true; saveWithCV(fieldValue, word);"
-      ng-repeat="word in profile.controlledVocabulary.dictionary | filter:displayVocabularyWord">
+      ng-repeat="word in profile.controlledVocabulary.dictionary | filter:displayVocabularyWord | orderBy:'name'">
```

Comment:
Are there other locations where `orderBy:'name'` sorting removed recently?

Reply:
See the PR referenced in the comments above.
Linked here for convenience:
- https://github.com/TexasDigitalLibrary/Vireo/pull/1851

Reply:
Yes, thank you. That is the PR I glanced at. That PR changed multiple template loop expressions. Do any of those need to be reverted back? Or just the one?

Reply:
Just this one as this PR states.